xbmc/utils/EGLUtils.cpp

```diff
@@ -210,15 +224,29 @@ bool CEGLContextUtils::ChooseConfig(EGLint renderableType, EGLint visualId)
   return true;
 }
 
-bool CEGLContextUtils::CreateContext(const EGLint* contextAttribs)
+bool CEGLContextUtils::CreateContext(CEGLAttributesVec contextAttribs)
 {
   if (m_eglContext != EGL_NO_CONTEXT)
   {
     throw std::logic_error("Do not call CreateContext when context has already been created");
   }
 
+  if (CEGLUtils::HasExtension(m_eglDisplay, "EGL_IMG_context_priority"))
```

Comment:
Why don't you `#ifdef EGL_IMG_context_priority` here instead of defining the stuff manually? Would look nicer imo.

Reply:
I have mixed opinions because I would have to ifdef twice. It's up to you if you want me to do so 👍

Reply:
IMO it's cleaner, but I won't insist. It's marked for removal anyway.